### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Admin/Pool.php [hunk 1]
```diff
@@ -20,22 +20,49 @@
  */
 class Pool
 {
+    /**
+     * @var ContainerInterface|null
+     */
     protected $container = null;
 
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Admin/Pool.php]: `string[]`

File: Admin/Pool.php [hunk 2]
```diff
@@ -20,22 +20,49 @@
...
+     */
     protected $adminServiceIds = array();
 
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Admin/Pool.php]: `string[]`

File: Admin/Pool.php [hunk 3]
```diff
@@ -20,22 +20,49 @@
...
+     */
     protected $adminGroups = array();
 
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Admin/Pool.php]: `string[]`

File: Admin/Pool.php [hunk 4]
```diff
@@ -20,22 +20,49 @@
...
+     */
     protected $adminClasses = array();
 
-    protected $templates    = array();
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Admin/Pool.php]: `string[]`

File: Admin/Pool.php [hunk 5]
```diff
@@ -20,22 +20,49 @@
...
+     */
+    protected $templates = array();
 
-    protected $assets       = array();
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Admin/Pool.php]: `string[]`

File: Controller/CRUDController.php [hunk 6]
```diff
@@ -248,12 +254,17 @@ public function listAction(Request $request = null)
         // set the theme for the current Admin Form
         $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
 
-        return $this->render($this->admin->getTemplate('list'), array(
-            'action'     => 'list',
-            'form'       => $formView,
-            'datagrid'   => $datagrid,
-            'csrf_token' => $this->getCsrfToken('sonata.batch'),
-        ), null, $request);
+        return $this->render(
```
Comment (Soullivaneuh) [inline Controller/CRUDController.php]: Please revert that.

BTW, this is not related to phpdoc PR.

File: Controller/CRUDController.php [hunk 7]
```diff
@@ -278,10 +289,12 @@ public function batchActionDelete(ProxyQueryInterface $query)
             $this->addFlash('sonata_flash_error', 'flash_batch_delete_error');
         }
 
-        return new RedirectResponse($this->admin->generateUrl(
-            'list',
-            array('filter' => $this->admin->getFilterParameters())
-        ));
+        return new RedirectResponse(
```
Comment (Soullivaneuh) [inline Controller/CRUDController.php]: Same here. Please revert.

File: Controller/CRUDController.php [hunk 8]
```diff
@@ -353,11 +366,16 @@ public function deleteAction($id, Request $request = null)
             return $this->redirectTo($object, $request);
         }
 
-        return $this->render($this->admin->getTemplate('delete'), array(
-            'object'     => $object,
-            'action'     => 'delete',
-            'csrf_token' => $this->getCsrfToken('sonata.delete'),
-        ), null, $request);
+        return $this->render(
+            $this->admin->getTemplate('delete'),
```
Comment (Soullivaneuh) [inline Controller/CRUDController.php]: I stop asking revert here, but the asking is for all your modifications.

File: Admin/AdminExtensionInterface.php [hunk 9]
```diff
@@ -107,9 +101,7 @@ public function alterNewInstance(AdminInterface $admin, $object);
      * Get a chance to modify object instance.
      *
      * @param AdminInterface $admin
-     * @param  $object
-     *
-     * @return mixed
+     * @param                $object
```
Comment (jvasseur) [inline Admin/AdminExtensionInterface.php]: You should add a type here (porbably mixed or object).

File: Command/GenerateAdminCommand.php [hunk 10]
```diff
@@ -300,6 +302,15 @@ private function askAndValidate(InputInterface $input, OutputInterface $output,
         return $questionHelper->ask($input, $output, $question);
     }
 
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @param                 $questionText
+     * @param                 $default
+     * @param                 $separator
```
Comment (jvasseur) [inline Command/GenerateAdminCommand.php]: Same here, missing a types (probably string).

File: Admin/Pool.php (hunk 2 above)
Comment (core23) [inline Admin/Pool.php]: This is an array with array values, no string array

File: Admin/Pool.php (hunk 3 above)
Comment (core23) [inline Admin/Pool.php]: This is an array with array values, no string array

File: Admin/Pool.php (hunk 5 above)
Comment (core23) [inline Admin/Pool.php]: Couldn't find any usage? Remove or keep?

File: Admin/Pool.php (hunk 2 above)
Comment (Soullivaneuh) [inline Admin/Pool.php]: Indeed, no usage. Oo

@rande Can you give us some clue?

File: Admin/Pool.php (hunk 5 above)
Comment (core23) [inline Admin/Pool.php]: @rande Can you give me some feedback? Could I add a deprecation info?

**Comment by core23** [comment]
@Soullivaneuh Can you merge this? I'll create a new PR for the deprecation in ``Pool``.

File: Admin/Pool.php (hunk 2 above)
Comment (rande) [inline Admin/Pool.php]: @Soullivaneuh There is an usage: https://github.com/core23/SonataAdminBundle/blob/phpdoc/Admin/Pool.php#L85-L96 and it is used to generate the side menu

File: Admin/Pool.php (hunk 5 above)
Comment (rande) [inline Admin/Pool.php]: It was used: https://github.com/sonata-project/SonataAdminBundle/commit/45b2ad31b50bacecc8eabf9a294e9e80889a7306 but some code has been removed.

Comment (rande) [inline Admin/Pool.php]: And removed here: https://github.com/sonata-project/SonataAdminBundle/commit/b4858a41f4e10507485359504720da8f9b6d8619#diff-ed85ead40da3b862ecc9ba97a2e62ac2 so you can remove the property.
